src/main/java/frc/robot/subsystems/intake/pivot/PivotCommandsBuilder.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.intake.pivot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+
+public class PivotCommandsBuilder {
+
+	private final Pivot pivot;
+
+	public PivotCommandsBuilder(Pivot pivot) {
+		this.pivot = pivot;
+	}
+
+	//@formatter:off
+	public Command goToPosition(Rotation2d position) {
+		return new FunctionalCommand(
+			() -> {},
+			() -> pivot.setPosition(position),
+			interrupted -> pivot.stayInPlace(),
+			() -> false,
+			pivot
+		).withName("Go to position: " + position.getDegrees());
+	}
+
+	public Command setPower(double power) {
```

Comment:
overload setPower to support DoubleSupplier

Reply:
no real need. in the code we will use go to position,  set power is basically only needed in testing